scripts/build-src-containers.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+echo "INFO: Build sources containers"
+if [[ -z "${REPODIR}" ]] ; then
+  echo "ERROR: REPODIR Must be set for build src containers"
+  exit 1
+fi
+
+BUILDSH=${REPODIR}/contrail-container-builder/containers/build.sh
+if ! [[ -x "${BUILDSH}" ]] ; then
+  echo "ERROR: build.sh tool from contrail-container-builder is not available in ${REPODIR} or is not executable"
+  exit 1
+fi
+
+PUBLISH_LIST_FILE=${PUBLISH_LIST_FILE:-"src_containers_to_publish"}
+if ! [[ -f "${PUBLISH_LIST_FILE}" ]] ; then
+  echo "ERROR: targets for build as src containers must be listed at ${PUBLISH_LIST_FILE}"
+  exit 1
+fi
+
+DOCKERFILE_TEMPLATE=scripts/Dockerfile.src.tmpl
+if ! [[ -f "${DOCKERFILE_TEMPLATE}" ]] ; then
+  echo "ERROR: Dockerfile template ${DOCKERFILE_TEMPLATE} is not available."
+  exit 1
+fi
+
+VENDOR_NAME=${VENDOR_NAME:-tungstenfabric}
```

Comment:
it is part of dev-env.. so it is possible to include ${my_dir}/../common/common.sh
and reuse defaults for variables